### PR TITLE
Add user.id to Sysmon's Event ID 22

### PR DESCRIPTION
**Pull request by janniten**
## What does this PR do?
Add the user.id field to EVENT ID 22 (from field winlog.user.identifier)
https://www.elastic.co/guide/en/ecs/current/ecs-user.html#field-user-id

## Why is it important?

During the revision of some windows detection rules I've found that Sysmon's event ID13 does not populate the user.id field, so those rules won't work.
The affected rule is External IP Lookup fron Non-Browser Process

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -164,6 +164,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Winlogbeat*
 
 - Add support for custom XML queries {issue}1054[1054] {pull}29330[29330]
+- Add user.id for Sysmon's Event ID 22 {pull]29660[29660]
```
Comment (efd6) [inline CHANGELOG.next.asciidoc:167]: Please fix the link syntax; `{pull}29660[29660]`.

Comment (janniten) [reply CHANGELOG.next.asciidoc]: @efd6 Fixed!

**Comment by janniten** [comment]
> This pull request is now in conflicts. Could you fix it? 🙏 To fixup this pull request, you can check out it locally. See documentation: https://help.github.com/articles/checking-out-pull-requests-locally/
> 
> ```
> git fetch upstream
> git checkout -b sysmon-evt-22-enrich upstream/sysmon-evt-22-enrich
> git merge upstream/master
> git push upstream sysmon-evt-22-enrich
> ```

@efd6 shall I have to execute this?

**Comment by efd6** [comment]
Given the size of this change you'll probably find it easier to just reset hard to the current master and replay the change manually in https://github.com/elastic/beats/blob/master/x-pack/winlogbeat/module/sysmon/ingest/sysmon.yml.

**Pull request closed without merging**